README.md

```diff
@@ -58,31 +58,29 @@ The `KnownTokenId` namespace manages the token IDs of supported cryptocurrencies
 
 Note: Users can transfer between different coins/tokens, not just the same token on different blockchains. For example, it's possible to convert BTC to WBTC when moving from Bitcoin to an EVM network.
 
-
 ### XLink SDK
-The `XLinkSDK` object contains the most important functions of this library, all grouped together. To create it:
+The [`XLinkSDK`](https://docs-typedoc.xlink-sdk.pages.dev/modules/XLinkSDK) object contains the most important functions of this library, all grouped together. To create it:
 
 ```typescript
 const theSdk = new XLinkSDK();
 ```
 
 For detailed API documentation, including a full list of available methods and their usage, please refer to:
 
-[Latest version SDK API Documentation](https://releases-latest.xlink-sdk.pages.dev/)
+[SDK API Documentation](https://docs-typedoc.xlink-sdk.pages.dev/classes/XLinkSDK.XLinkSDK)
 
-### USE CASES
+### Use Cases
 
 Create an instance of the SDK with default options
 ```typescript
-import{ XLinkSDK } from '@xlink-project/xlink-sdk/src';
-
+import{ XLinkSDK } from '@xlink-network/xlink-sdk';
 const xlinkSdk = new XLinkSDK();
 ```
 1. Bridge from Stacks
 ```typescript
-import { BridgeInfoFromStacksInput } from '@xlink-project/xlink-sdk/src/xlinkSdkUtils/bridgeInfoFromStacks';
-import { BridgeFromStacksInput } from '@xlink-project/xlink-sdk/src/xlinkSdkUtils/bridgeFromStacks';
-import { KnownChainId, KnownTokenId } from '@xlink-project/xlink-sdk/src/utils/types/knownIds';
+import { BridgeInfoFromStacksInput } from '@xlink-network/xlink-sdk/xlinkSdkUtils/bridgeInfoFromStacks';
```

Comment:
Users will not be able to import content from `/xlinkSdkUtils/bridgeInfoFromStacks` as we have set up the ECMA module info in the package.json

Reply:
we'll check it (I guess you're right), but it will be required when the btc part be implemented right?
We can leave the btc usage section out by the moment.

Reply:
> required when the btc part be implemented right?

do you mean `/bitcoinHelpers` ? yes, we will need it, but not `import { selectUTXOs } from "@xlink-network/xlink-sdk/bitcoinUtils/selectUTXOs" , it will be `import { selectUTXOs } from "@xlink-network/xlink-sdk/bitcoinHelpers`

because of the definitions here:

https://github.com/xlink-network/xlink-sdk/blob/0df7c76f159161d0e059ac69397b28a4673cce9a/package.json#L23-L32